modules/nuxt-ngrok.yml

```diff
@@ -0,0 +1,16 @@
+name: nuxt-ngrok
```

Comment:
I would recommend using just the `ngrok` slug rather than `nuxt-ngrok`.

Reply:
I saw there is `ngrok` already (for nuxt v2) and the `autofix.ci` was editing that file if the name was ngrok

Reply:
I think it would be worth renaming that one instead so the maintained module has the simple slug. You will need to update the slug and the file name.

Reply:
okay, what would you suggeste to name the old one? thanks

Reply:
it could be nuxt2-ngrok?